config/routes.rb

```diff
@@ -28,6 +28,7 @@
       resource :assignment_override, only: :create
 
       # Server-side authenticated endpoints
+      resources :split_details, only: :show, controller: 'split_details'
```

Comment:
I don't think we need the `controller:` argument here. Does it not work without it?